### PR TITLE
feat — changed background, page layout

### PR DESCRIPTION
**Pull request by pavloshaposhnikov**

### CODE REVIEW

File: src/modules/home/components/Intro/Intro.module.scss [hunk 1]
```diff
@@ -1,59 +1,68 @@
...
-		margin-top: 65px;
 		border-radius: 40px;
+		width: 80%;
 	}
 
 	img {
 		display: block;
-		width: 100%;
 		height: auto;
+		width: 100%;
```
Comment (purr-purr) [inline src/modules/home/components/Intro/Intro.module.scss:71]: display: block;
  height: auto;
  width: 100%;
  max-height: 400px;

File: src/modules/home/components/Intro/Intro.module.scss [hunk 2]
```diff
@@ -1,59 +1,68 @@
...
 	}
 }
 
 .info {
-	@include flex(center, center, 29px);
+	@include flex(start, center, 50px);
+	flex-direction: column;
+	text-align: left;
+	width: 50%;
+	padding-top: 120px;
```
Comment (purr-purr) [inline src/modules/home/components/Intro/Intro.module.scss]: remove padding-top and center content

File: src/modules/home/components/Intro/Intro.module.scss [hunk 3]
```diff
@@ -1,59 +1,68 @@
...
 	}
 }
 
 .poster {
-	width: 100%;
-	max-height: 550px;
 	border-radius: 70px;
 	overflow: hidden;
-	margin-top: 130px;
+	width: 50%;
```
Comment (purr-purr) [inline src/modules/home/components/Intro/Intro.module.scss]: only   for poster
```
border-radius: 70px;
overflow: hidden; 
```

File: src/modules/home/components/Intro/Intro.module.scss (hunk 1 above)
Comment (pavloshaposhnikov) [reply src/modules/home/components/Intro/Intro.module.scss:71]: done

File: src/modules/home/components/Intro/Intro.module.scss (hunk 2 above)
Comment (pavloshaposhnikov) [reply src/modules/home/components/Intro/Intro.module.scss]: done

File: src/modules/home/components/Intro/Intro.module.scss (hunk 3 above)
Comment (pavloshaposhnikov) [reply src/modules/home/components/Intro/Intro.module.scss]: done

**Pull request merged**